src/components/CommentInput/index.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { FC } from 'react';
+import { StyleSheet, TextInput, View, ViewStyle } from 'react-native';
+
+import { useInput } from '../../utils/hook';
+
+interface Props {
+	style?: ViewStyle;
+	containerStyle?: ViewStyle;
+}
+
+export const CommentInput: FC = ({
+	style = {},
+	containerStyle = {},
+}: Props) => {
```

Comment:
this`Props` should be in `CommentInput: FC<Props>` instead.

Reply:
I've fixed it, pls check again